### PR TITLE
NDS Cleanup

### PR DESCRIPTION
**Pull request by bobwise**
- convert long inline code to block code in Installation Guide
- "[BUG] (not a stopper) - weird horizontal scrolling iOS safari, this is because of the long code"
- "In the Angular guide, I think this should be "installing""
- "We call it "your-app" in one spot and "you-app" in another. This should be consistent."
- "This attribute is required..." is in here twice. Remove one."
- "[BUG] the "Getting Started Guides" link on https://spark-docs.netlify.com/installing-spark/, just reloads the current page"
- "should be a link to that section"
- update css for code in links [`like` this]()

### CODE REVIEW

File: src/pages/installing-spark.mdx [hunk 1]
```diff
@@ -28,4 +28,4 @@ Spark in various states of implementation.
 |---|---|
 | `master` | The Spark Design System and all of its prerequisites are installed. Start building with Spark components right away. |
 | `kitchen-sink` | The Spark Design System and all of its prerequisites are installed. In addition, a test page has been added that includes all the Spark components. This is useful for testing the latest version of Spark. |
-| `without-spark` | This is a boilerplate app without Spark installed. This is useful for following along in our [Getting Started Guides](). |
+| `without-spark` | This is a boilerplate app without Spark installed. This is useful for following along in our [Getting Started Guides](./installing-spark/html-installation). |
```
Comment (afebbraro) [inline src/pages/installing-spark.mdx]: These links shouldnt be './'

Comment (bobwise) [reply src/pages/installing-spark.mdx]: when I take out the './' they break. What should this link be?

Comment (bobwise) [reply src/pages/installing-spark.mdx]: i fixed it

File: src/scss/_spark-theme.scss [hunk 2]
```diff
@@ -20,13 +20,17 @@ $sprk-webfonts:
...
+}
+
 #gatsby-focus-wrapper code {
   line-height: 1;
   white-space: nowrap;
   font-size: 13px;
   color: rgba(51, 51, 51, 0.9);
   background-color: rgb(248, 248, 248);
-  margin: 0px 2px;
+  margin: 0px 0px;
```
Comment (RCopeland) [inline src/scss/_spark-theme.scss]: should just do margin: 0; here

File: src/pages/installing-spark.mdx [hunk 3]
```diff
@@ -28,4 +28,4 @@ Spark in various states of implementation.
 |---|---|
 | `master` | The Spark Design System and all of its prerequisites are installed. Start building with Spark components right away. |
 | `kitchen-sink` | The Spark Design System and all of its prerequisites are installed. In addition, a test page has been added that includes all the Spark components. This is useful for testing the latest version of Spark. |
-| `without-spark` | This is a boilerplate app without Spark installed. This is useful for following along in our [Getting Started Guides](). |
+| `without-spark` | This is a boilerplate app without Spark installed. This is useful for following along in our [Getting Started Guides](./html-installation). |
```
Comment (RCopeland) [inline src/pages/installing-spark.mdx]: i think you need /installing-spark/html-installation here instead of ./html-installation

**Comment by bobwise** [comment]
requests should be fixed now

**Comment by afebbraro** [comment]
@bobwise everything looks great! 🔥 thank you for doing this!

**Pull request merged**